packages/plugins/plugin-registry/src/components/PluginList.tsx

```diff
@@ -15,9 +15,11 @@ export type PluginListProps = Omit<PluginItemProps, 'plugin'> & {
 
 export const PluginList = ({ plugins = [], ...props }: PluginListProps) => {
   return (
-    <List classNames='grid grid-cols-[repeat(auto-fill,minmax(240px,1fr))] gap-3 p-3'>
+    <List classNames='grid grid-cols-[repeat(auto-fill,minmax(300px,1fr))] auto-rows-[10rem] gap-3 p-3 overflow-x-hidden overflow-y-auto scrollbar-thin'>
```

Comment:
What is `scrollbar-thin`? I think this doesn’t exist… is it a Tailwind v4 feature?

Reply:
see `tailwind-scrollbar` in react-ui-theme